src/main/test/com/epam/preprod/sirenko/services/impl/SerializationServiceImplTest.java

```diff
@@ -0,0 +1,78 @@
+package com.epam.preprod.sirenko.services.impl;
+
+import com.epam.preprod.sirenko.entity.Clothing;
+import com.epam.preprod.sirenko.entity.DryFood;
+import com.epam.preprod.sirenko.entity.Food;
+import com.epam.preprod.sirenko.entity.Product;
+import com.epam.preprod.sirenko.enums.PetGroup;
+import com.epam.preprod.sirenko.enums.Season;
+import com.epam.preprod.sirenko.enums.Size;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.*;
+import java.math.BigDecimal;
+import java.util.ArrayList;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class SerializationServiceImplTest {
```

Comment:
Maybe we should rename this class because in current implementation we dont test SerializationServiceImpl class here. We just serealize products with by default and gzip

Reply:
agree, changed